assets/src/amp-story-editor-blocks.js

```diff
@@ -100,6 +151,10 @@ const addAMPAttributes = ( settings, name ) => {
 		};
 	}
 
+	// Disable anchor support as we auto-generate IDs.
+	settings.supports = settings.supports || {};
```

Comment:
Is it safe to mutate `supports` directly? Should it not be extended:

```suggestion
	settings = Object.assign( {}, settings );
	settings.supports = Object.assign( {}, settings.supports );
```

Reply:
I might actually rewrite it a bit to use object spread instead, makes the code more consistent too.